test/wasm/assets/012_partialobjects.yaml

```diff
@@ -8,7 +8,8 @@ cases:
         p["b"] = 2
     want_defined: true
   - note: additive (negative)
-    query:  'data.x.p = {"a": 1, "b": 2}'
+    query: 'data.x.p = input'
```

Comment:
The typing has become a bit stricter -- this query would fail type-checking now instead of failing evaluation. By shovelling the object into input, we sidestep that.